packages/types/workflow/index.ts

```diff
@@ -0,0 +1,9 @@
+import Step from '../step';
+/**
+ * Workflow is the definition holder for a set of Steps that can become a Task
+ */
+interface Workflow {
+  id: string;
+  steps: Step[];
```

Comment:
I would love for us to use some flow control api. Sets of steps here is making lots of assumptions. Imagine that steps flow can be defined and there will be StepsFlow class.
![http://www.clinicalhomecare.co.uk/images/stories/ncha-images/ncha-flow-diagram.png](http://www.clinicalhomecare.co.uk/images/stories/ncha-images/ncha-flow-diagram.png)

Then by default we can have Simple flow that will just have array there.

Some things to consider:
- https://github.com/substack/node-seq